sa/database.go

```diff
@@ -80,13 +71,33 @@ func NewDbMapFromConfig(config *mysql.Config, maxOpenConns int) (*gorp.DbMap, er
 		// In MariaDB, max_statement_time and long_query_time are both seconds.
 		// Note: in MySQL (which we don't use), max_statement_time is millis.
 		readTimeout := config.ReadTimeout.Seconds()
-		_, err := dbmap.Exec("SET SESSION max_statement_time = ?, long_query_time = ?;",
+		prefix = fmt.Sprintf(
+			"SET STATEMENT max_statement_time=%g, long_query_time=%g, sql_mode='STRICT_ALL_TABLES' FOR ",
 			readTimeout*0.95, readTimeout*0.80)
-		if err != nil {
-			return nil, err
-		}
 	}
 
+	// Choose a random driver name to avoid conflicts.
+	driverNum, err := rand.Int(rand.Reader, big.NewInt(math.MaxInt64))
```

Comment:
I mainly did this because I would expect any auditors* looking at our code to raise their eyebrows at the use of non-crypto rand. While reading the code would reveal that it's fine, I think it's better to just always use crypto rand in Boulder.

*including ourselves